packages/client/src/NetworkNodeFacade.ts

```diff
@@ -142,31 +159,30 @@ export class NetworkNodeFacade {
      * Subsequent calls to getNode/start will fail.
      */
     private destroy = pOnce(async () => {
-        const node = this.cachedNode
+        const network = this.cachedNode
```

Comment:
Why this change? Or should we rename some other variables/fields/methods too, e.g. `this.cachedNode`?